Formula/v/vulkancapsviewer.rb

```diff
@@ -0,0 +1,30 @@
+class Vulkancapsviewer < Formula
+  desc "Client application to display hardware implementation details for Vulkan GPUs"
+  homepage "https://github.com/SaschaWillems/VulkanCapsViewer"
+  url "https://github.com/SaschaWillems/VulkanCapsViewer.git", tag: "3.32", revision: "3024e3148a7b47706b6d259f8e0ad0a2aa651330"
+  license "LGPL-3.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "vulkan-headers" => :build
+  depends_on "qt@5"
+  depends_on "vulkan-loader"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args,
+                    "-DQt5_DIR=#{Formula["qt5"].opt_lib}/cmake/Qt5",
+                    "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].opt_prefix}",
+                    "-DVULKAN_LOADER_INSTALL_DIR=#{Formula["vulkan-loader"].opt_prefix}"
+    system "cmake", "--build", "build"
+    if OS.mac?
+      bin.install "build/vulkanCapsViewer.app/Contents/MacOS/vulkanCapsViewer"
+    else
+      bin.install "build/vulkanCapsViewer"
+    end
+  end
+
+  test do
+    with_env("QT_QPA_PLATFORM" => "offscreen") do
```

Comment:
```suggestion
   ENV["QT_QPA_PLATFORM"] = "offscreen"
```

Reply:
That leaks the environment modification, but sure since we are running this single command, not anything else, no harm done.